client/displayservers/X11/x11.c

```diff
@@ -23,6 +23,7 @@
 #include "x11.h"
 #include "atoms.h"
 #include "clipboard.h"
+#include "../../../resources/icondata.h"
```

Comment:
Instead of this reverse traversal please make the following change and just `#include "resources/icondata.h"`

```patch
diff --git a/client/CMakeLists.txt b/client/CMakeLists.txt
index ac0bef16..b21278bd 100644
--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -107,6 +107,7 @@ add_custom_command(
 )
 
 include_directories(
+       ${PROJECT_TOP}
        ${PROJECT_SOURCE_DIR}/include
        ${CMAKE_BINARY_DIR}/include
 )
```

Reply:
all set. i had a feeling you wouldn't like that ugly reverse traversal.